codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/RestJsonProtocolGeneratorTests.kt

```diff
@@ -255,7 +255,10 @@ class RestJsonProtocolGeneratorTests {
                             useFIPS: useFIPS
                         )
                     }
-            
+
+                    public var partitionID: String? {
+                        return ["ExampleClient", region].compactMap { ${'$'}0 }.joined(separator: " - ")
```

Comment:
trivial: this can be interpolated string.

Reply:
@ganeshnj This is the test expectation... I presume you mean to use an interpolated string [here](codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSServiceConfig.kt)?

Reply:
I changed this to interpolation.  In the event `region` is not set here (per our discussion, that is unlikely), there will just be a trailing ` - ` on the partition ID string.  This is fine because this string is not user-facing, only used for uniqueness.